### PR TITLE
[hi] Update data/i18n/hi/hi.toml

### LINKED ISSUE #31013: [hi] Update data/i18n/hi/hi.toml to include missing strings
**This is a Bug Report**

**Problem:**
The current [data/i18n/hi/hi.toml](https://github.com/kubernetes/website/blob/dev-1.22-hi.1/data/i18n/hi/hi.toml) on branch `dev-1.22-hi.1` is not up to date with the English version.

**Proposed Solution:**
Compare [en.toml](https://github.com/kubernetes/website/blob/dev-1.22-hi.1/data/i18n/en/en.toml) with [hi.toml](https://github.com/kubernetes/website/blob/dev-1.22-hi.1/data/i18n/hi/hi.toml) and add the missing strings (19 in number)

**Page to Update:**
https://github.com/kubernetes/website/blob/dev-1.22-hi.1/data/i18n/hi/hi.toml

**Note:**
The current working branch for `hi` localization work is `dev-1.22-hi.1`. Please make sure to open a PR against `dev-1.22-hi.1`.
For more info see Hindi localization progress tracking issue #29353

/language hi
/kind bug

### PR DESCRIPTION
**Pull request by Babapool**
Fixes #31013 

This PR updates the [hi.toml](https://github.com/kubernetes/website/blob/dev-1.22-hi.1/data/i18n/hi/hi.toml) by adding the 19 missing strings to make it at par with the [en.toml](https://github.com/kubernetes/website/blob/dev-1.22-hi.1/data/i18n/en/en.toml).

Page updated:
https://github.com/kubernetes/website/blob/dev-1.22-hi.1/data/i18n/hi/hi.toml

(This is my first PR for the Hindi localization, please share any feedback so that I can improve for next time.)

Signed by Vitthal Sai : vitthalsai2001@gmail.com
/language hi

### CODE REVIEW

**Comment by AvineshTripathi** [comment]
Hey thanks for the contribution I have done the review and LGTM from myside

also looking in the file I see the `hi.toml` is differently organised in comparison to `en.toml` is this done intentionally was curious to know

File: data/i18n/hi/hi.toml
Comment (sftim) [inline data/i18n/hi/hi.toml:247]: nit: I'd localize “content guide“.

**Review by sftim — COMMENTED** [review]
Hi. Please try to have this file in alphabetical order (fixing that order could be a second commit) by the translation key.

The first entry should be `[caution]`.

File: data/i18n/hi/hi.toml
Comment (Babapool) [reply data/i18n/hi/hi.toml:247]: Thanks @sftim , I will do the needful

**Comment by Babapool** [comment]
@AvineshTripathi I have no idea on how the hi.toml was arranged as compared to en.toml. It is better to ask the maintainers and reviewers on this issue. Thank for the LGTM.

**Comment by Babapool** [comment]
@sftim I have seen how `ja.toml` , `ko.toml` and `zh.toml` are arranged. I am thinking to arrange `hi.toml` alphabetically wise mirroring `en.toml` (in English alphabetical order).  If more people agree on arranging order I am happy to push another commit to rearrange the file. @anubha-v-ardhan @divya-mohan0209

**Comment by anubha-v-ardhan** [comment]
@Babapool agreed, we should arrange it in alphabetical order.

**Comment by Babapool** [comment]
Sure thing @anubha-v-ardhan , I will do the needful(arranging in English alphabetical order). Also can you suggested should [`Content Guide`](https://github.com/kubernetes/website/pull/31072#discussion_r774072888) be translated or transliterated? Thanks

**Comment by divya-mohan0209** [comment]
> Sure thing @anubha-v-ardhan , I will do the needful(arranging in English alphabetical order). Also can you suggested should [`Content Guide`](https://github.com/kubernetes/website/pull/31072#discussion_r774072888) be translated or transliterated? Thanks

I'd rather that it be translated if possible.

File: data/i18n/hi/hi.toml
Comment (divya-mohan0209) [inline data/i18n/hi/hi.toml:250]: For versions, although "संस्करणों" does pop up in Google translate I don't think it is the plural form that is commonly used. 
When we say other versions/different versions, it is always referred to as 
अन्य संस्करण as opposed to संस्करणों.
```suggestion
other = "संस्करण"
```

File: data/i18n/hi/hi.toml
Comment (divya-mohan0209) [reply data/i18n/hi/hi.toml:247]: तीसरे पक्ष की परियोजनाओं  is a literal transliteration. We're looking at third party in the context of these projects/products not being developed/maintained by the K8s community. The below suggestion fits better, IMO. WDYT?

```suggestion
other = """यह खंड अन्य पक्ष परियोजनाओं से जुड़ा है जो कुबेरनेट्स द्वारा आवश्यक कार्यक्षमता प्रदान करते हैं। कुबेरनेट्स परियोजना के लेखक इन परियोजनाओं के लिए जिम्मेदार नहीं हैं। यह पृष्ठ <a href="https://github.com/cncf/foundation/blob/master/website-guidelines.md" target="_blank">CNCF website guidelines</a> का अनुसरण करते हुए परियोजनाओं को वर्णानुक्रम में सूचीबद्ध करता है। इस सूची में कोई नयी परियोजना जोड़ने से पहले यह <a href="/docs/contribute/style/content-guide/#third-party-content">content guide</a> पृष्ट पढ़के ही परिवर्तन करें।"""
```

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
Grammatical nits.

File: data/i18n/hi/hi.toml
Comment (Babapool) [reply data/i18n/hi/hi.toml:247]: Yes I think using `अन्य` will be much better and it gives a clear idea of the context. I will the do the needful and push a new commit based on the feedback. Thank you.

**Comment by Babapool** [comment]
Hi all PTAL on the new changes. @divya-mohan0209 I am still confused on what to use for `Content Guide`  please suggest on it. Thanks.

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
@Babapool 
Some strings are still missing. It would be nice if you compare this with [en.toml](https://github.com/kubernetes/website/blob/main/data/i18n/en/en.toml) and add the missing ones

**Comment by Babapool** [comment]
@anubha-v-ardhan Sure I will do the needful. Thanks

**Comment by Babapool** [comment]
Hi @anubha-v-ardhan PTAL at the new changes I have pushed. Thanks (and Happy New Years Eve and Happy New Year in advance to all).

File: data/i18n/hi/hi.toml
Comment (anubha-v-ardhan) [inline data/i18n/hi/hi.toml:241]:
```suggestion
other = """यह खंड अन्य पक्ष परियोजनाओं से जुड़ा है जो कुबेरनेट्स द्वारा आवश्यक कार्यक्षमता प्रदान करते हैं। कुबेरनेट्स परियोजना के लेखक इन परियोजनाओं के लिए जिम्मेदार नहीं हैं। यह पृष्ठ <a href="https://github.com/cncf/foundation/blob/master/website-guidelines.md" target="_blank">CNCF वेबसाइट दिशानिर्देश</a> का अनुसरण करते हुए परियोजनाओं को वर्णानुक्रम में सूचीबद्ध करता है। इस सूची में कोई नई परियोजना जोड़ने से पहले यह <a href="/docs/contribute/style/content-guide/#third-party-content">विषय मार्गदर्शक</a> पृष्ट पढ़के ही परिवर्तन करें।"""
```

File: data/i18n/hi/hi.toml
Comment (anubha-v-ardhan) [inline data/i18n/hi/hi.toml:250]:
```suggestion
other = """<p>इस पृष्ठ की वस्तुओं अन्य पक्ष के उत्पादों या परियोजनाओं से जुड़ा है जो कुबेरनेट्स द्वारा आवश्यक कार्यक्षमता प्रदान करते हैं। कुबेरनेट्स परियोजना के लेखक इन अन्य पक्ष के उत्पादों या परियोजनाओं के लिए जिम्मेदार नहीं हैं। अधिक जानकारी के लिए यह <a href="https://github.com/cncf/foundation/blob/master/website-guidelines.md" target="_blank">CNCF वेबसाइट दिशानिर्देश</a> पृष्ठ पढ़े।"</p><p>कोई नई अतिरिक्त अन्य पक्ष लिंक जोड़ने से पहले यह पृष्ठ <a href="/docs/contribute/style/content-guide/#third-party-content">विषय मार्गदर्शक</a> पृष्ठ पढ़के ही परिवर्तन का प्रस्ताव प्रस्तुत करें।</p>"""
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
I think we can use `विषय मार्गदर्शक` for content guide. It sounds much simpler and easier to understand.

File: data/i18n/hi/hi.toml
Comment (Babapool) [reply data/i18n/hi/hi.toml]: I have added the fix for `Content Guide` as suggested. PTAL. Thanks.

**Comment by Babapool** [comment]
I have added the fix for Content Guide as suggested. PTAL. Thanks. @anubha-v-ardhan @divya-mohan0209

**Review by divya-mohan0209 — COMMENTED** [review]
I don't know if we should transliterate issue or if we can leave it as is. Thoughts, @sftim or @anubha-v-ardhan ?

**Comment by anubha-v-ardhan** [comment]
> I don't know if we should transliterate issue or if we can leave it as is. Thoughts, @sftim or @anubha-v-ardhan ?

I think we can leave it as it is

**Comment by Babapool** [comment]
Hi all, do I need to do something else from my side regarding the PR.? Thanks.

**Comment by divya-mohan0209** [comment]
@Babapool : I think you need to retain the word issue instead of transliterating it.

**Comment by Babapool** [comment]
@divya-mohan0209: Just to confirm by meaning to retain the word Issue  you mean to change/replace ` इशू` by `issue` ?

**Comment by Babapool** [comment]
Hi all,  PTAL. Thanks. @anubha-v-ardhan @divya-mohan0209

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
LGTM from me
@divya-mohan0209 Is this PR an LGTM from you too?

**Comment by anubha-v-ardhan** [comment]
@Babapool Can you squash the commits, I think we can merge this

**Comment by Babapool** [comment]
@anubha-v-ardhan I will try to merge the commits

**Comment by Babapool** [comment]
Hi @anubha-v-ardhan I have squashed the commits into one. PTAL. Thanks.
/assign @anubha-v-ardhan

**Comment by anubha-v-ardhan** [comment]
/assign
/lgtm
/approve

Thank you @Babapool :)

**Pull request merged**
